README.md

```diff
@@ -28,7 +28,7 @@ def long_time_func(a: int, b: int, c: int) -> int:
 def long_time_func_2(n_tuple: tuple, power: int) -> int:
     return [number ** power for number in n_tuple]
 
-long_time_func(1, 2, 3)
+long_time_func(1, 2, 3) 
```

Comment:
Restore changes

Reply:
What changes exactly need to be restored in this fragment?